core/src/bin/test_oidc_call_back.rs

```diff
@@ -0,0 +1,28 @@
+use mongo_odbc_core::oidc_auth::*;
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() {
```

Comment:
What are you testing here?
My understanding is that you are checking that:
- You can authenticate, both when there is no refresh token and when there is one.
- After the first oidc call, you get back a refresh token and it can be used in subsequent call

If this correct, please, add a short comment on top of your function.

Also, what about using an expired refresh token. Could we add a case for it?

Reply:
I added a test with a bad refresh token, which works as expected. The idp is actually not configured for refresh tokens to ever expire, only access tokens expire.

Reply:
Ah, yes, I couldn't remember if there was an expiration or not. I know it is possible, but if ours don't have one, then all good :)